### ISSUE TITLE
Documentation request for latest changes

### ISSUE THREAD

**Issue by janissimsons** [Unity-Technologies/2d-extras#47]
Hi,
@ChuanXin-Unity could you please describe in details what has changed in your latest commit and how should we be using the scripts?
Also, if it's possible, it would be nice to have a little more info on how to use the GridInformation script.
Thank you

**Comment by ChuanXin-Unity** [comment]
Yes, we will update the documentation with more details, including usage of GridInformation.
